conf/layer.conf

```diff
@@ -8,3 +8,4 @@ BBFILES += "${LAYERDIR}/recipes-*/*/*.bb \
 BBFILE_COLLECTIONS += "validation"
 BBFILE_PATTERN_validation = "^${LAYERDIR}/"
 BBFILE_PRIORITY_validation = "6"
+LAYERSERIES_COMPAT_yoctobsp = "rocko"
```

Comment:
yoctobsp? It should match your layer name, not some other layer's name

Reply:
arf sorry. I made a copy paste error